### DISCUSSION TITLE
为什么我自己训练的kenlm模型无法加载

### DISCUSSION THREAD

**Discussion by socialwork123** [shibing624/pycorrector#538] [Q&A]
from pycorrector import Corrector
model = Corrector(language_model_path='kongjian.klm')
print(model.correct('非法正据'))，我将zh_giga.no_cna_cmn.prune01244.klm文件删除，希望他用我自己训练的kongjian.klm,结果他又从网上下载了zh_giga.no_cna_cmn.prune01244.klm，最后还是用这个模型进行了纠错，我该怎么办？

**Comment by shibing624** [comment]
指定了Corrector(language_model_path='kongjian.klm')，优先用自己的模型做纠错的，看下指定的路径是否正确，写绝对路径吧。
